### PR TITLE
Add app store assets, add billing options, update docs

### PR DESCRIPTION
**Pull request by polishq**
#### What problem is this solving?

Update app so that it can be offered on the VTEX App Store.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -8,18 +8,19 @@
...
+Reviews & Ratings is a VTEX IO native solution allowing shoppers to submit and view reviews and ratings for products.
 
 ![reviews-and-ratings-app](https://user-images.githubusercontent.com/52087100/71026526-31e7d580-20e8-11ea-93d8-094c1e8af7cd.png)
 
 ## Configuration
 
-1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) the `reviews-and-ratings` app in the desired account;
-2. In the admin's account, access **Apps** and then select the **Reviews and Ratings** box;
+1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) `vtex.reviews-and-ratings` in the desired account;
+
```
Comment (guerreirobeatriz) [inline docs/README.md:18]:
```suggestion
1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) `vtex.reviews-and-ratings@1.x` in the desired VTEX account;
```

File: docs/README.md [hunk 2]
```diff
@@ -8,18 +8,19 @@
...
+2. In the admin dashboard, access `Apps > My Apps` and then click the box for `Reviews and Ratings`;
```
Comment (guerreirobeatriz) [inline docs/README.md:19]:
```suggestion
2. In the account's admin, access `Apps > My Apps` and then click the box for `Reviews and Ratings`;
```

File: docs/README.md [hunk 3]
```diff
@@ -31,7 +32,7 @@ The Reviews and Ratings app is a VTEX native solution for product review and rat
 
 - **Default all review accordions to open**: Checking this box will cause all review accordions on the product page to default to open with review text limited to 3 lines. Reviews with more than 3 lines of text will be truncated with and ellipsis and a "Show More" link that can be used to display the whole review text.
 
-:warning: This app fills the standard VTEX review blocks with content using abstract interfaces from `vtex.product-review-interfaces` . The **VTEX review blocks** are:
+4. Place the following blocks in your store-theme:
 
 - `"product-reviews"`: This block can be added to the product page (`store.product`). It renders a paginated list of reviews for the product being viewed, as well as a form for the shopper to add a new review.
```
Comment (guerreirobeatriz) [inline docs/README.md:37]: These blocks have props?

File: docs/README.md (hunk 1 above)
Comment (guerreirobeatriz) [inline docs/README.md:18]:
```suggestion
1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) `vtex.reviews-and-ratings@1.x` in the desired account;
```

File: docs/README.md (hunk 2 above)
Comment (guerreirobeatriz) [inline docs/README.md:19]:
```suggestion
2. In the account's admin dashboard, access `Apps > My Apps` and then click the box for `Reviews and Ratings`;
```

File: docs/README.md (hunk 3 above)
Comment (polishq) [reply docs/README.md:37]: Nope, no props

**Pull request merged**